### DISCUSSION TITLE
Multi-tenancy in ABP

### DISCUSSION THREAD

**Discussion by naveengarla** [abpframework/abp#9014] [Q&A]
I was trying to implement Multitenancy architecture using ABP Framework, but somehow unable to find any solution in the given samples code where they implemented **each tenant has a separate, dedicated database** to store the data related to that tenant. Please let me know anyone tried this implementation using ABP Framework. If possible provide a documentation link for reference.

I see documentation for [tenant management module](https://docs.abp.io/en/abp/latest/Modules/Tenant-Management) is still incomplete.

**Comment by maliming** [comment]
You can take a look at this https://github.com/abpframework/abp/issues/7655#issue-803588821
